### PR TITLE
remove unused cmake environment variable from path

### PR DESCRIPTION
**Pull request by ericLemanissier**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ script:
   - conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
   - conan install . --build=missing
   - source activate.sh
-  - PATH=$CMAKE_PREFIX_PATH/bin:$PATH
```
Comment (MartinDelille) [inline .travis.yml:13]: Removing this instruction lead to the use of the `qmake` version in the path that use Qt 4.8.7 as you can see here: https://travis-ci.org/MartinDelille/conan-qt-example/builds/555015588#L1903

Comment (MartinDelille) [reply .travis.yml:13]: Anyway I would have considered right that the `activate.sh` update the `PATH` environment variable so I don't have to do it myself.

Comment (ericLemanissier) [reply .travis.yml:13]: Ok. You are right that activate.sh should make package's qmake in PATH. Maybe it appends to PATH instead of prepending?

Comment (MartinDelille) [reply .travis.yml:13]: If it appends to PATH and a `qmake` is already in the path (like qmake for Qt 4.8.7), the version from the conanfile.txt will be ignored.

Comment (MartinDelille) [reply .travis.yml:13]: I noticed that on windows, 'activate.bat' update the 'PATH'.

**Comment by ericLemanissier** [comment]
superseded by #3

**Pull request closed without merging**